src/main/java/build/buildfarm/worker/shard/FuseExecFileSystem.java

```diff
@@ -74,8 +80,10 @@ public Path createExecDir(
       Map<build.bazel.remote.execution.v2.Digest, Directory> directoriesIndex,
       DigestFunction.Value digestFunction,
       Action action,
-      Command command)
+      Command command,
+      UserPrincipal owner)
       throws IOException, InterruptedException {
+    // FIXME insist that the FUSE perform permission checks against the owner
```

Comment:
question: do we need a GH Issue to track this?

Reply:
Much bigger issue overall: what to do with the FUSE impl. It would be great, but needs to be rewritten (leaks inodes in mounted filesystem). This is a note for the future, when I come through here and think that this already had permission checks.